business_requirement/views/business_view.xml

```diff
@@ -260,7 +260,7 @@
         <field name="name">Categories</field>
         <field name="res_model">business.requirement.category</field>
         <field name="view_mode">tree,form,graph</field>
-        <field name="view_type">form</field>
+        <field name="binding_view_types">form</field>
```

Comment:
You must remove this, as this is for binding an action in the dropdown menu "Actions".